### ISSUE TITLE
nested forms js problem in more than 1 level

### ISSUE THREAD

**Issue by evgemar** [Kunstmaan/KunstmaanBundlesCMS#560]
There is problem with adding 1st level objects, when page form has multiple nesting.

In the PageType:
```
        $builder->add('questions', 'collection', [
            'type' => new QuestionType(),
            'required'  => false,
            'allow_add'    => true,
            'allow_delete'    => true,
            'delete_empty'  => true,
            'by_reference' => false,
            'attr' => [
                'nested_form' => true,
                'nested_form_min' => 0,
                'nested_form_max' => 40,
            ],
            'label' => 'questions'
        ]);
```
In the QuestionType:
```
        $builder->add('answers', 'collection', array(
            'type' => new AnswerType(),
            'required' => false,
            'allow_add'    => true,
            'allow_delete'  => true,
            'delete_empty'  => true,
            'by_reference' => false,
            'attr' => array(
                'nested_form' => true,
                'nested_form_min' => 0,
                'nested_form_max' => 40,
            ),
            'label' => 'answers'
        ));
```
When you try to add new question the embeded form will be added on the page form container and on each of previous questions forms.

**Comment by mlebkowski** [comment]
I believe it’s fixed in bd4281a3cb6a59e3189cddccb25d0312d24a806e, could you check with the latest dev version?

**Comment by krispypen** [comment]
This is fixed in 3.2.1
